docs/configuration/configuration.md

```diff
@@ -1724,6 +1724,10 @@ url: <string>
 headers:
   [ <string>: <string> ... ]
 
+# Retry on receiving a rate-limiting status code 429, in response from remote-write storage.
+# This is expected to change at any point or be removed in future releases.
```

Comment:
I'd suggest alternate wording, and explictely mark this as experimental.

```suggestion
# Retry upon receiving a 429 status code from the remote-write storage.
# This is expected to change at any point or be removed in future releases.
```

Reply:
Done